From c2120c639a9874d37fad84bdf71eb2eb9eddb75e Mon Sep 17 00:00:00 2001
From: Daniil Krant <daniil.krant@globallogic.com>
Date: Tue, 20 Mar 2018 12:23:46 +0200
Subject: [PATCH] Configure RPI to use USB audio sink as a default

Signed-off-by: Daniil Krant <daniil.krant@globallogic.com>
---
 asound.conf | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/asound.conf b/asound.conf
index 03653cf..05890af 100644
--- a/asound.conf
+++ b/asound.conf
@@ -1 +1,12 @@
 # Global alsa-lib configuration
+
+pcm.!default {
+    type hw
+    card 1
+}
+
+ctl.!default {
+    type hw
+    card 1
+}
+
-- 
2.7.4

