From d104462744c171db9fa021aff331a96f7633c530 Mon Sep 17 00:00:00 2001
From: Daniil Krant <daniil.krant@globallogic.com>
Date: Tue, 24 Apr 2018 12:46:30 +0300
Subject: [PATCH] [audio] Add support of stereo headsets

Jira-Id: ICS-1302

Signed-off-by: Daniil Krant <daniil.krant@globallogic.com>
---
 asound.conf | 28 ++++++++++++++++++++++++----
 1 file changed, 24 insertions(+), 4 deletions(-)

diff --git a/asound.conf b/asound.conf
index 6d346d2..f673d80 100644
--- a/asound.conf
+++ b/asound.conf
@@ -1,11 +1,31 @@
 # Global alsa-lib configuration
 
-pcm.!default {
-    type hw
-    card 1
+pcm.record_mono {
+    type dsnoop
+    ipc_key 234884
+    slave {
+        pcm "hw:1,0"
+        channels 1
+    }
+    bindings.0  0
+}
+pcm.capture_dev {
+    type route
+    slave.pcm record_mono
+    ttable {
+        0.0 1
+        1.1 0
+    }
 }
 
-ctl.!default {
+pcm.playback_dev {
     type hw
     card 1
 }
+
+pcm.!default {
+    type asym
+    playback.pcm "playback_dev"
+    capture.pcm "capture_dev"
+}
+
-- 
2.7.4

